### PR TITLE
Support stacks

### PR DESCRIPTION
**Pull request by rberrelleza**

### CODE REVIEW

**Review by pchico83 — APPROVED** [review]
Cool!
We also need one for helm install/uninstall

File: deploy-stack/action.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+description: "Deploys a stack in Okteto Cloud"
+inputs:
+  namespace: 
+    description: 'If present, the namespace scope for this action. If not specified, it will use the current context'
+    required: false
+  name: 
+    description: "The name of the stack. If missing, it will use the one in your okteto-stack.yml file"
+    required: false
+  build: 
+    description: "Create the deployment if it doesn't exist in the namespace. If not specified, the action will fail if the deployment doesn't exist"
```
Comment (pchico83) [inline deploy-stack/action.yml]: The description is wrong no? It should say build images

File: deploy-stack/entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+if [ "$build" == "true" ]; then
+params="${params} --build"
+fi
+
+if [ ! -z "$wd" ]; then
+cd $wd
+fi
+
+echo running: okteto stack deploy $params on $(pwd)
+okteto stack deploy $params
```
Comment (pchico83) [inline deploy-stack/entrypoint.sh:28]: you can add `--wait`

File: deploy-stack/action.yml (hunk 1 above)
Comment (rberrelleza) [reply deploy-stack/action.yml]: yeah, copy paste error

File: deploy-stack/entrypoint.sh (hunk 2 above)
Comment (rberrelleza) [reply deploy-stack/entrypoint.sh:28]: it's the base param, line 9.

**Pull request merged**